### ISSUE TITLE
declare(strict_types=1);

### ISSUE THREAD

**Issue by devnix** [underground-works/sublime-php-namespace-monkey#5]
The possibility to add a `declare(strict_types=1)` would literally save me more seconds each day!

```php
<?php

declare(strict_types=1);

namespace Acme\Foo;

class Bar
```

Love your plugin since the first day 😄

**Comment by itsgoingd** [comment]
Released 0.4 with an option to include the strict types declaration (needs to be enabled in settings). You can start thinking about what you will do with all that newly saved time. 🙂

**Comment by devnix** [comment]
Yes! 
![image](https://user-images.githubusercontent.com/1777519/105813485-2f74d200-5fb0-11eb-882e-6e285363e6d6.png)

I'll spend the time [here](https://github.com/devnix/chipmunkphp), and probably thanking you 😄
